siad/main.go

```diff
@@ -132,6 +132,9 @@ Explorer (e):
 
 // main establishes a set of commands and flags using the cobra package.
 func main() {
+	if build.DEBUG {
+		fmt.Println("Running with debugging enabled")
+	}
```

Comment:
I had a hunch that on Linux, it should be possible to access the binary of a currently-running process, even if the original binary was deleted. And indeed, it is possible, though it takes some work.

1. Get the PID of the running process
2. Open `/proc/$pid/maps` and locate the line containing the path of the original binary
3. Copy the inode number from that line (last column before the path)
4. Get the device descriptor of the filesystem containing the inode
5. Run `debugfs -w $device -R 'ln <$inode> /path/to/file` (may require `sudo`)

The binary is now available at `/path/to/file`. (An important quirk is that the path is relative to the device descriptor mount. So if you had `/home` mounted on `/dev/sda3`, the file will show up in `/home/path/to/file`.)

Pretty cool. Also pretty impractical :P